Source/WebKit/Shared/Extensions/wpe/WebExtensionUtilitiesWPE.cpp

```diff
@@ -28,6 +28,7 @@
 
 #if ENABLE(WK_WEB_EXTENSIONS)
 
+#include "WPEDisplay.h"
 #include "WPEToplevelPrivate.h"
```

Comment:
You should not add the wpe platform headers directly, there's a single header `<wpe/wpe-platform.h>` that should be included conditionally using  `#if ENABLE(WPE_PLATFORM)`